studies/migrations/0060_auto_20200219_1036.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
In the future - to avoid having too many migrations per PR, one thing I do is roll back my changes by migrating to the previous migration and then re-create the files so that it "squashes" the migration actions into one file. You can think of it like a `git reset --soft HEAD~${NUM_OF_COMMITS_BACKWARD}`

Reply:
Oh, good point, thanks!